iOS/chronreact/AppDelegate.m

```diff
@@ -18,51 +19,31 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   [BatchPush enableAutomaticDeeplinkHandling:NO];
+  
+  // Print out the Batch installation id for testing
```

Comment:
No still figuring out what it is, I just upgraded the react version and got rid of redundant code.